Project.toml

```diff
@@ -34,25 +34,25 @@ Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
 DiffEqBase = "6.36"
 DiffResults = "0.0.4, 1.0"
 Distances = "0.8, 0.9, 0.10"
-Distributions = "0.21, 0.22, 0.23, 0.24, <0.25.11"
+Distributions = "0.21, 0.22, 0.23, 0.24, 0.25"
 DocStringExtensions = "0.8"
 DynamicHMC = "2.1, 3"
 ForwardDiff = "0.10"
 LabelledArrays = "0.7, 1.0"
 LogDensityProblems = "0.9, 0.10, 0.11"
 MacroTools = "0.5"
 Missings = "0.4, 1.0"
-ModelingToolkit = "5.6, 6"
+ModelingToolkit = "5.6, 6, 8"
 Optim = "0.19, 0.20, 0.21, 0.22, 1.0"
 PDMats = "0.9, 0.10, 0.11"
 Parameters = "0.12"
 RecursiveArrayTools = "1,2"
 Reexport = "0.2, 1.0"
 Requires = "0.5, 1.0"
-StanSample = "3, 4"
+StanSample = "3, 4, 6"
 StructArrays = "0.4, 0.5, 0.6"
-TransformVariables = "0.3, 0.4"
-Turing = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17"
+TransformVariables = "0.3, 0.4, 0.6"
+Turing = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.21"
```

Comment:
We should drop some old versions here since many of those wouldn't ever be used on v1.6